.github/workflows/.reusable-deploy-ecs.yml

```diff
@@ -61,7 +62,7 @@ jobs:
           build-args: CI_COMMIT_SHA=${{ github.sha }}
           secrets: |
             github_private_cloud_token=${{ secrets.GH_PRIVATE_ACCESS_TOKEN }}
-            sse_pgp_pkey=${{ secrets.SSE_PGP_PRIVATE_KEY }}
+            "sse_pgp_pkey=${{ secrets.SSE_PGP_PRIVATE_KEY }}"
```

Comment:
Odd that it's only this one that needs quotations. Could / should we add a comment?

Reply:
The reason for the quotes is that the value is multiline.

Reply:
A comment would be tough because yaml `|` makes it impossible to comment the same line.

Reply:
Ok, yep, fair point. Hopefully this conversation will serve enough if people are searching for a reason in the future...